index.html

```diff
@@ -123,15 +123,15 @@
 This list is far from exhaustive.
 Often, there are multiple reasons that a site seeks to identify a visitor.
 
-Once someone has provided an identifier,
-sites are technically able to use identifiers in any way they choose.
+Once someone has provided an their identity document,
+sites are technically able to use information from such documents in any way they choose.
```

Comment:
I think that this really does need to be "identifier".  Though the DC work is the motivation for this, I would prefer that we make a broader statement about identifiability instead.  That's a stronger point to be making.

Reply:
Having reverted the changes, I think the document already makes a pretty broad statement around identifiability... but we can file a bug for that if you'd like.